generators/app/index.js

```diff
@@ -424,7 +424,7 @@ module.exports = class extends BaseGenerator {
             this.rewriteFile(
               adminModulePath,
               'jhipster-needle-add-admin-route',
-              ',\n      {\n        path: \'entity-audit\',\n        loadChildren: () => import(\'./entity-audit/entity-audit.module\').then(m => m.EntityAuditModule)\n      }'
+              '      {\n        path: \'entity-audit\',\n        loadChildren: () => import(\'./entity-audit/entity-audit.module\').then(m => m.EntityAuditModule)\n      }'
```

Comment:
Keep comma for next needle.
```suggestion
              '      {\n        path: \'entity-audit\',\n        loadChildren: () => import(\'./entity-audit/entity-audit.module\').then(m => m.EntityAuditModule)\n      },'
```

Reply:
@mshima It is already adding it, but I do not know how:

![image](https://user-images.githubusercontent.com/3706415/97111301-fb022900-16d5-11eb-96d5-e6bde6704a80.png)

Reply:
Prettier is adding.
But it will fail if add a second needle is added.

Reply:
@mshima Will regenerate a new app and confirm

Reply:
It's a module, prettier probably will be executed before any other interaction.